app/src/ui/merge-branch/merge.tsx

```diff
@@ -320,9 +283,7 @@ export class Merge extends React.Component<IMergeProps, IMergeState> {
           />
         </DialogContent>
         <DialogFooter>
-          {enableMergeConflictDetection()
-            ? this.renderNewMergeInfo()
-            : this.renderOldMergeMessage()}
+          {this.renderNewMergeInfo()}
```

Comment:
Should this be renamed to `renderNewMergeInfo`?

Reply:
@say25 yes please!